### ISSUE TITLE
Not availble on Linux

### ISSUE THREAD

**Issue by Spodeian** [Kaedrin/warhammer-mod-manager#67]
I understand this was probably designed for windows, but could you either compile an executable for Linux (Ubuntu in my case), or if you are willing, actually make the source code available and then I can try and then people can try and do it for them selves at least.
If this just straight up isn't viable on Linux, then fair enough, I'd like to know tho.

**Comment by Goldfive** [comment]
Yeah I'm having this issue as well. The CA launcher is currently limiting me, but Kaedrin won't even startup because it won't find the path to the exe. It's possible to change it in the GameData.xml, but haven't had any success with that yet.

**Comment by dabanhfreak** [comment]
Currently running Warhammer 2 on Proton. When launching the mod manager, it says that Steam is not active.
Obviously it's because it's not detecing the Linux steam launcher... perhaps if there is a script that can be made to patch this, that'll be great.
